src/sentry/static/sentry/app/actionCreators/dashboards.tsx

```diff
@@ -0,0 +1,92 @@
+import {Client} from 'app/api';
+import {t} from 'app/locale';
+import {addErrorMessage} from 'app/actionCreators/indicator';
+import {
+  DashboardListItem,
+  OrgDashboardResponse,
+  OrgDashboard,
+  OrgDashboardUpdate,
+} from 'app/views/dashboardsV2/types';
+
+export function createDashboard(
+  api: Client,
+  orgId: string,
+  newDashboard: DashboardListItem
+): Promise<OrgDashboardResponse> {
+  const promise: Promise<OrgDashboardResponse> = api.requestPromise(
+    `/organizations/${orgId}/dashboards/`,
+    {
+      method: 'POST',
+      data: {title: newDashboard.title},
```

Comment:
Should we send the widgets as well? The dashboard could be created with an initial set of widgets as the user could make a new dashboard add some widgets and then save it.

Reply:
We should. It doesn't looks like the API endpoint for `POST /organization/:orgId/dashboards/` supports this yet.